Add missing dispose() methods

<!--

* Please fill out this template with all the relevant information so we can
  understand what's going on and fix the issue. We appreciate bugs filed and PRs
  submitted!

* You can get the installed version of an NPM package by running `npm ls <insert package name>` in your terminal.

-->

-   `three` version: 0.130.1
-   `node` version: v14.17.2
-   `npm` version: 6.14.13

### Problem description:

<!-- Please describe why the current behaviour is a problem -->
It looks like some dispose() functions are missing.

### Relevant code:

<!-- feel free to input the code in the space below -->

```ts
const axesHelper = new AxesHelper(5)
axesHelper.dispose()

 error TS2339: Property 'dispose' does not exist on type 'AxesHelper'.
```

### Suggested solution:
<!--
It's ok if you don't have a suggested solution, but it really helps if you could
do a little digging to come up with some suggestion of how to improve things.
-->
Add dispose() function in `helpers/AxesHelper.d.ts` (and probably elsewhere).
See also: https://github.com/mrdoob/three.js/pull/21577

--- a/.all-contributorsrc
+++ b/.all-contributorsrc
@@ -205,6 +205,15 @@
       "contributions": [
         "code"
       ]
+    },
+    {
+      "login": "servinlp",
+      "name": "Servin Nissen",
+      "avatar_url": "https://avatars.githubusercontent.com/u/7222029?v=4",
+      "profile": "https://github.com/servinlp",
+      "contributions": [
+        "code"
+      ]
     }
   ],
   "skipCi": true,

--- a/README.md
+++ b/README.md
@@ -51,6 +51,7 @@ Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/d
     <td align="center"><a href="https://bjornstar.com/"><img src="https://avatars.githubusercontent.com/u/20630?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Bjorn Stromberg</b></sub></a><br /><a href="https://github.com/three-types/three-ts-types/commits?author=bjornstar" title="Code">💻</a></td>
     <td align="center"><a href="https://github.com/happy-turtle"><img src="https://avatars.githubusercontent.com/u/18415215?v=4?s=100" width="100px;" alt=""/><br /><sub><b>HappyTurtle</b></sub></a><br /><a href="https://github.com/three-types/three-ts-types/commits?author=happy-turtle" title="Code">💻</a></td>
     <td align="center"><a href="https://studio.did0es.me/"><img src="https://avatars.githubusercontent.com/u/38882716?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Shuta Hirai</b></sub></a><br /><a href="https://github.com/three-types/three-ts-types/commits?author=shuta13" title="Code">💻</a></td>
+    <td align="center"><a href="https://github.com/servinlp"><img src="https://avatars.githubusercontent.com/u/7222029?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Servin Nissen</b></sub></a><br /><a href="https://github.com/three-types/three-ts-types/commits?author=servinlp" title="Code">💻</a></td>
   </tr>
 </table>
 

--- a/types/three/src/helpers/AxesHelper.d.ts
+++ b/types/three/src/helpers/AxesHelper.d.ts
@@ -13,4 +13,6 @@ export class AxesHelper extends LineSegments {
     type: string;
 
     setColors(xAxisColor: Color, yAxisColor: Color, zAxisColor: Color): this;
+
+    dispose(): void;
 }

--- a/types/three/src/helpers/CameraHelper.d.ts
+++ b/types/three/src/helpers/CameraHelper.d.ts
@@ -13,4 +13,6 @@ export class CameraHelper extends LineSegments {
     type: string;
 
     update(): void;
+
+    dispose(): void;
 }

--- a/types/three/src/lights/Light.d.ts
+++ b/types/three/src/lights/Light.d.ts
@@ -64,4 +64,6 @@ export class Light extends Object3D {
      * @deprecated Use shadow.mapSize.height instead.
      */
     shadowMapHeight: any;
+
+    dispose(): void;
 }

--- a/types/three/src/lights/LightShadow.d.ts
+++ b/types/three/src/lights/LightShadow.d.ts
@@ -62,4 +62,5 @@ export class LightShadow {
     updateMatrices(light: Light, viewportIndex?: number): void;
     getViewport(viewportIndex: number): Vector4;
     getFrameExtents(): Vector2;
+    dispose(): void;
 }

